src/plugin.json

```diff
@@ -73,6 +78,224 @@
       "path": "datasource/plugin.json"
     }
   ],
+  "roles": [
+    {
+      "role": {
+        "name": "Checks reader",
+        "description": "Read checks in the Synthetic Monitoring app",
+        "permissions": [
+          { "action": "plugins.app:access", "scope": "plugins:id:grafana-synthetic-monitoring-app" },
+          { "action": "grafana-synthetic-monitoring-app:read" },
```

Comment:
What does this action provide beyond the above plugin access?

Reply:
Same answer as in https://github.com/grafana/synthetic-monitoring-app/pull/982#discussion_r1853908817